src/components/sitepages/SiteData.vue

```diff
@@ -199,16 +202,21 @@ export default {
     return {
       accordionIsOpen: 1,
 
-      activeImageToolsTab: 1, // default tab to set: controls / analysis / data / dev
+      activeImageToolsTab: '', 
       activeAnalysisTab: 'star inspector',  // default tab in 'analysis'
       activeDevTab: 'recents3',  // default tab in 'dev tools'
 
       region_stats_loading: false,
       image_stats_loading: false,
     }
   },
-  methods: {
 
+  mounted() {
+    //default image tools tab, set to analysis in site_config.js
+    this.activeImageToolsTab = this.selected_image_tools_tab; 
```

Comment:
Where is `this.selected_image_tools_tab` defined? I don't see any other references to it in the file.

Reply:
Might not be clear enough from the comment above it - another store value defined (currently) in the site_config.js file